packages/core/src/progress-circle/progress-circle.element.scss

```diff
@@ -21,6 +21,7 @@ svg {
 .private-host {
   @include center-content('block');
   @include equilateral(100%);
+  filter: brightness(95%);
```

Comment:
This is a temp fix I think, Colin and I are going to make an audit through the themes for any other potential issues I may have missed.

Reply:
What is adding a brightness filter attempting to fix?

Reply:
There is a contrast issue with the background of the ring, it's slightly too light. This kind of was a bandaid fix, but I'm cool with removing it until I get a proper fix in with the theme status colors.

Reply:
I see. Does the ring's background track matter that much? I would think the contrast of the foreground fill/rotation would be what a11y cares about most. The background track is decorative and doesn't convey any information whatsoever.

Reply:
Agree, going to remove for now